src/shared/components/match/match-score.tsx

```diff
@@ -18,20 +18,60 @@ interface MatchScoreProps {
 export const MatchScore = React.memo<MatchScoreProps>(
   ({ match, children }: MatchScoreProps) => {
     const { formatDate, formatTime } = useDate();
+    const today = new Date();
+    const tomorrow = new Date();
+    tomorrow.setDate(today.getDate() + 1);
+    const yesterday = new Date();
+    yesterday.setDate(today.getDate() - 1);
```

Comment:
You can use `dayUtils` instead https://github.com/wanned/karmine-corp-app/blob/b4082dd734def341f6ed4da5edc5453025acd214/src/shared/utils/days.ts#L3